tests/providers/google/cloud/operators/test_gcs_to_gcs_operator_system.py

```diff
@@ -16,33 +16,70 @@
 # specific language governing permissions and limitations
 # under the License.
 """System tests for Google Cloud Build operators"""
+import pytest
 
-
-from tests.providers.google.cloud.operators.test_gcs_to_gcs_system_helper import GcsToGcsTestHelper
+from airflow.providers.google.cloud.example_dags.example_gcs_to_gcs import (
+    BUCKET_1_DST, BUCKET_1_SRC, BUCKET_2_DST, BUCKET_2_SRC, BUCKET_3_DST, BUCKET_3_SRC,
+)
 from tests.providers.google.cloud.utils.gcp_authenticator import GCP_GCS_KEY
-from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, provide_gcp_context, skip_gcp_system
-from tests.test_utils.system_tests_class import SystemTest
+from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest, provide_gcp_context
+
+
+@pytest.mark.backend("mysql", "postgres")
+@pytest.mark.system("google.cloud")
+@pytest.mark.credential_file(GCP_GCS_KEY)
+class GcsToGcsExampleDagsSystemTest(GoogleSystemTest):
```

Comment:
Can the GoogleSystemTest automatically apply the system mark? Does that work?

Reply:
I think it would not work (I think markers are not , but something that is worth checking @nuclearpinguin :)

Reply:
If `GoogleSystemTest` is marked with system test then all inheriting classes will be marked as well.